README.md

```diff
@@ -228,149 +230,148 @@ Any contribution is welcome!
-* [MakeHuman](http://www.makehumancommunity.org/) - Parametric human character modeling
-* [mmg](https://github.com/MmgTools/mmg) - mmg is an open source software for bidimensional and tridimensional surface and volume remeshing
-* [Open3D](http://www.open3d.org/) - A Modern Library for 3D Data Processing
-* [OpenSubdiv](http://graphics.pixar.com/opensubdiv/docs/intro.html) - A set of open source libraries that implement high performance subdivision surface evaluation on massively parallel CPU and GPU architectures
-* [Optimesh](https://github.com/nschloe/optimesh) - Python library to optimise a triangular mesh, remesh with various 
-* [Partio](https://www.disneyanimation.com/technology/partio.html) - A unified interface that makes it easier to load, save, and manipulate particle files
-* [PCL](https://pointclouds.org/) - A standalone, large scale, open project for 2D/3D image and point cloud processing
-* [Polygon Mesh Processing Library](https://www.pmp-library.org/) - A modern C++ open-source library for processing and visualizing polygon surface meshes
-* [QHull](https://github.com/qhull/qhull/wiki) - Convex hull generator
-* [Quadriflow](https://github.com/hjwdzh/QuadriFlow) - Convert a mesh to a quadrangulation algorithms 
-* [Rigify](https://github.com/LesFeesSpeciales/rigify) - Auto-rigging Addon for Blender
-* [Simplex](https://github.com/blurstudio/Simplex) - Blendshape Solver
-* [SOFA](https://www.sofa-framework.org/) - An efficient framework dedicated to research, prototyping and development of physics-based simulations
-* [Polyscope](https://github.com/nmwsharp/polyscope) - A C++ & Python viewer for 3D data like meshes and point clouds
+* [BlenderGIS](https://github.com/domlysz/BlenderGIS) - Geo data importer.
+* [CGAL](https://www.cgal.org/) - A software project that provides easy access to efficient and reliable geometric algorithms in the form of a C++ library.
+* [CvWrap](https://github.com/chadmv/cvwrap) - Fast Maya wrap deformer.
+* [Dem Bones](https://www.ea.com/seed/news/open-source-dem-bones) - A library for skinning decomposition.
+* [Easy3D](https://github.com/LiangliangNan/Easy3D) - A lightweight, easy-to-use, and efficient C++ library for processing and rendering 3D data .
+* [Eigen](http://eigen.tuxfamily.org/index.php?title=Main_Page) - A C++ template library for linear algebra: matrices, vectors, numerical solvers, and related algorithms.
+* [Embree](https://www.embree.org/) - A collection of high performance ray tracing kernels that helps graphics application engineers to improve the performance of their photorealistic rendering application.
+* [Geometry Central](https://geometry-central.net/) - A modern C++ library of data structures and algorithms for geometry processing, with a particular focus on surface meshes.
+* [Instant Meshes](https://github.com/wjakob/instant-meshes) - Remesher.
+* [autoremesher](https://github.com/huxingyi/autoremesher) -  Automatic quad remeshing tool.
```

Comment:
I think alphabetical is best for being partial and for finding links when you know the spelling of a tool. Otherwise it looks like we play favorites if we order it freestyle.